store/src/main/java/org/apache/rocketmq/store/timer/TimerMessageStore.java

```diff
@@ -632,22 +637,23 @@ public boolean enqueue(int queueId) {
                     int sizePy = bufferCQ.getByteBuffer().getInt();
                     bufferCQ.getByteBuffer().getLong(); //tags code
                     MessageExt msgExt = getMessageByCommitOffset(offsetPy, sizePy);
+                    currQueueOffset = offset + (i / CQ_STORE_UNIT_SIZE);
```

Comment:
It's recommend to use CQ_STORE_UNIT_SIZE or ConsumeQueue.CQ_STORE_UNIT_SIZE in all occurrence.

Reply:
Why relocate the line from the end of the loop to this position?

Reply:
The isRunningEnqueue method checkpoint the offset when the resource role changes. The offset in this case should be nextOffset

Reply:
@drpmma @zhouxinyu help review it

Reply:
@RongtongJin help review it